frontend/src/components/core/Block/Block.tsx

```diff
@@ -281,6 +278,7 @@ class Block extends PureComponent<Props> {
           {...widgetProps}
         />
       ),
+      oldTable: (el: SimpleElement) => <OldTable element={el} width={width} />,
```

Comment:
... This should not get into production.

Reply:
It won't :)